pkgs/development/tools/symfony-cli/default.nix

```diff
@@ -12,12 +12,20 @@ buildGoModule rec {
     sha256 = "sha256-GAsyI8I+tHFMV/LqwPx2ph+w3zaqKSn9vieVQcuO+y0=";
   };
 
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+  ];
+
   postInstall = ''
     mv $out/bin/symfony-cli $out/bin/symfony
   '';
 
   # Tests requires network access
-  doCheck = false;
+  checkPhase = ''
+    $GOPATH/bin/symfony-cli
```

Comment:
When overriding `checkPhase`, you should also provide `runHook preCheck` and `runHook postCheck`. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md> for more info.

Though, in this case, it might be better to use `passthru.tests` instead of replacing `checkPhase` so that it can be verified it works outside of the build environment (in a cleaner environment, more representative of actual users’ systems). And also so that we can re-instate `doCheck` when we manage to fix it.

https://github.com/NixOS/nixpkgs/blob/8bc6f346d4d17469205ab23ec92f0222311fc354/pkgs/tools/misc/osm2pgsql/default.nix#L49-L51

Reply:
@jtojnar I can provide an updated PR but I would need more info on this.

Reply:
What info do you mean?